NeurodataWithoutBorders/matnwb
# Fix multidimensional equality checks for arrays

Fixes #487 
## Motivation

A cleaner redo instead of coercing the type's shape.

## How to test the behavior?

1) Set up a valid `FeatureExtraction` Object
2) set its `features` property to some randomized value
3) All values of any type should pass. `single` typed values should not be converted.

## Checklist
